src/Zygote.jl

```diff
@@ -35,6 +35,7 @@ include("lib/nnlib.jl")
 include("lib/broadcast.jl")
 include("lib/forward.jl")
 include("lib/utils.jl")
+include("lib/fft.jl")
```

Comment:
I think this file is short enough that it could just go in `array.jl`, after linear algebra.

Reply:
I moved it to array.jl now